campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -286,6 +387,78 @@
   {% if blockable %}
     </div>
   {% endif %}
+  {% if background_image %}
+    <div class="heart-background-image-loader">
+      <div class="heart-background-image-left-container">
+        <div class="heart-background-row" style="width: 680px;">
+          <img src="{{ background_image }}"/>
+          <img src="{{ background_image }}"/>
+          <img src="{{ background_image }}"/>
+        {% if special_image_first %}
```

Comment:
Turns out the outdated code, which I just fixed in a new pull request, added the fifth image, and also moved them around a bit.